metal/mmtl/metal_model.py

```diff
@@ -114,6 +114,7 @@ def forward(self, X, task_names):
 
             head_module = self.head_modules[task_name].module
             if head_module not in outputs:
+                outputs[middle_module]["data"] = outputs[middle_module]["data"][1]
```

Comment:
BERT output is as a tuple, (encoded_layers, pooled_output). Since we only want a classifier, we only want the pooled output, which is the second element in this tuple. Hence, we make out input data for the linear layer the second element.